source/common/upstream/outlier_detection_impl.cc

```diff
@@ -248,6 +248,13 @@ void DetectorImpl::onConsecutive5xxWorker(HostSharedPtr host) {
     return;
   }
 
+  // There are two outcomes here. The ejection will be enforced,
```

Comment:
I see as well :). Updated the comment. I think this one flows a bit more clearly. Let me know.